src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.Designer.cs

```diff
@@ -0,0 +1,1611 @@
+﻿// <auto-generated />
```

Comment:
Could you please still add a file header?

Reply:
added